### PR TITLE
feat: added header to the html

### PR DESCRIPTION
**Pull request by Ben2oid**
Added header

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  </head>
```
Comment (hegedus-mark) [inline index.html:7]: interesting

**Comment by ErikRabCC** [comment]
Nice work, could you please make the text blue?

File: index.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+  <body>
+    <header><h1>Hello world</h1></header>
```
Comment (hegedus-mark) [inline index.html]: I don't like this message

**Review by hegedus-mark — CHANGES_REQUESTED** [review]
I suggested some changes

**Comment by Ben2oid** [comment]
> Nice work, could you please make the text blue?

Ye, bruv, I can make it blue.

**Comment by Ben2oid** [comment]
I fixed some stuff. Letters are now blue

**Comment by ErikRabCC** [comment]
Nice work, but styling belongs to the css file, can you make it not inline?

**Comment by Ben2oid** [comment]
> Nice work, but styling belongs to the css file, can you make it not inline?

sure thing buddy

**Comment by hegedus-mark** [comment]
> > Nice work, but styling belongs to the css file, can you make it not inline?
> 
> sure thing buddy

I can quote reply I think

**Comment by ErikRabCC** [comment]
Nice work, I think it is ready

**Comment by Ben2oid** [comment]
Hey everyone. I think this feature is done, can we go ahead and merge?

**Comment by ErikRabCC** [comment]
Yes bro, go for it

**Comment by kristofNyikes** [comment]
Sure go ahead

**Comment by prvics** [comment]
> Hey everyone. I think this feature is done, can we go ahead and merge?

🫡

**Comment by pixhy** [comment]
merge it

**Review by hegedus-mark — APPROVED** [review]
I'm still not satisfied, but I approve

**Pull request merged**